### PR TITLE
revert-fees-from-SCRs

### PR DESCRIPTION
**Pull request by sasurobert**
fix revert fees

### CODE REVIEW

**Review by iulianpascalau — DISMISSED** [review]

**Review by iulianpascalau — DISMISSED** [review]

File: integrationTests/mock/intermediateTransactionHandlerMock.go [hunk 1]
```diff
@@ -21,10 +21,11 @@ type IntermediateTransactionHandlerMock struct {
 }
 
 // RemoveProcessedResults -
-func (ith *IntermediateTransactionHandlerMock) RemoveProcessedResults() {
+func (ith *IntermediateTransactionHandlerMock) RemoveProcessedResults() [][]byte {
 	if ith.RemoveProcessedResultsCalled != nil {
 		ith.RemoveProcessedResultsCalled()
```
Comment (SebastianMarian) [inline integrationTests/mock/intermediateTransactionHandlerMock.go:26]: return ith.RemoveProcessedResultsCalled()

File: process/mock/intermProcessorStub.go [hunk 2]
```diff
@@ -21,10 +21,11 @@ type IntermediateTransactionHandlerStub struct {
 }
 
 // RemoveProcessedResults -
-func (ith *IntermediateTransactionHandlerStub) RemoveProcessedResults() {
+func (ith *IntermediateTransactionHandlerStub) RemoveProcessedResults() [][]byte {
 	if ith.RemoveProcessedResultsCalled != nil {
 		ith.RemoveProcessedResultsCalled()
```
Comment (SebastianMarian) [inline process/mock/intermProcessorStub.go:26]: return ith.RemoveProcessedResultsCalled()

File: process/mock/intermediateTransactionHandlerMock.go [hunk 3]
```diff
@@ -22,10 +22,11 @@ type IntermediateTransactionHandlerMock struct {
 }
 
 // RemoveProcessedResults -
-func (ith *IntermediateTransactionHandlerMock) RemoveProcessedResults() {
+func (ith *IntermediateTransactionHandlerMock) RemoveProcessedResults() [][]byte {
 	if ith.RemoveProcessedResultsCalled != nil {
 		ith.RemoveProcessedResultsCalled()
```
Comment (SebastianMarian) [inline process/mock/intermediateTransactionHandlerMock.go:27]: return ith.RemoveProcessedResultsCalled()

File: integrationTests/mock/intermediateTransactionHandlerMock.go (hunk 1 above)
Comment (sasurobert) [reply integrationTests/mock/intermediateTransactionHandlerMock.go]: done

File: process/mock/intermProcessorStub.go (hunk 2 above)
Comment (sasurobert) [reply process/mock/intermProcessorStub.go]: done

File: process/mock/intermediateTransactionHandlerMock.go (hunk 3 above)
Comment (sasurobert) [reply process/mock/intermediateTransactionHandlerMock.go]: done

**Review by SebastianMarian — APPROVED** [review]

**Review by iulianpascalau — APPROVED** [review]

**Review by gabi-vuls — APPROVED** [review]
System test passed.

**Pull request merged**